nservicebus/hosting/index.md

```diff
@@ -44,6 +44,10 @@ NOTE: In Version 6, the endpoint instance is not disposable due to the asynchron
 
 A [Windows Service](https://msdn.microsoft.com/en-us/library/d56de412.aspx) is the most common way NServiceBus is hosted.
 
+**Stopping Windows Service**
```

Comment:
AFAIK this content is specific to the NSB host and should exist here http://docs.particular.net/nservicebus/hosting/nservicebus-host/  ?